src/compoz.ts

```diff
@@ -360,6 +367,10 @@ export class Compoz {
 			}
 		}
 
+		this.elInput.addEventListener("input", () => {
```

Comment:
Per my comment on this PR and T9696, I think we should not use change listener for creating draft.

Reply:
Okay Kak @shuLhan, now I use the change listener for undo & redo action. Please check the changes